src/marqo/tensor_search/add_docs.py

```diff
@@ -195,8 +196,8 @@ def download_and_chunk_media(url: str, device: str, modality: Modality,
     )
 
     if processor.total_size > MAX_FILE_SIZE:
-        raise ValueError(
-            f"File size ({processor.total_size / 1024 / 1024:.2f} MB) exceeds the maximum allowed size of 100 MB")
+        raise S2InferenceError(
```

Comment:
Good fix! The error will be caught and only return 400 for the specific document now, instead of failing the whole batch, right? 

Also, create something like the `MediaExceedsMaxSize` error and raise it here.

Reply:
Yep thats's right this will allow document only to fail not batch

Reply:
Addressed